src/Conticket/ApiBundle/Controller/EventController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Conticket\ApiBundle\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+use FOS\RestBundle\Controller\FOSRestController;
+use FOS\RestBundle\Controller\Annotations;
+use FOS\RestBundle\Request\ParamFetcherInterface;
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use FOS\RestBundle\Util\Codes;
+
+use Conticket\ApiBundle\Handler\EventHandler;
+use Conticket\ApiBundle\Document\Event;
+use Conticket\ApiBundle\Form\Type\EventType;
+
+final class EventController extends FOSRestController implements ClassResourceInterface
+{
+    /* @var Conticket\ApiBundle\Handler\EventHandler */
+    private $handler;
+    
+    public function __construct(EventHandler $handler) 
+    {
+        $this->handler = $handler;
+    }
+    
+    /**
+     * List all events.
+     *
+     * @return array
+     */
+    public function cgetAction()
```

Comment:
I can't see looking at this code, what methods are mandatory implemented by the interface or overwrite by Inheritance. Can you add `{@inheritdoc}` annotation on that methods